.storybook/vanillaish.js

```diff
@@ -3,33 +3,15 @@ import { create } from "@storybook/theming/create";
 export default create({
   base: "light",
 
-  colorPrimary: "#fff",
-  colorSecondary: "rgba(0,0,0,0.05)",
-
   // UI
-  appBg: "#fff",
-  appContentBg: "#fff",
-  appBorderColor: "#cdcdcd",
   appBorderRadius: ".125rem",
```

Comment:
I'm not sure what it does, but we probably don't need that either. We are removing all rounded corners from Vanillla in 4.0.

Reply:
This is for the controls - without, it is even rounder. I think it is better to keep it.